### PR TITLE
drop support for ruby < 1.9.3

### PR DESCRIPTION
**Pull request by koriroys**

### CODE REVIEW

**Comment by yob** [comment]
Is it possible to keep support for 1.9.3 for now? It's only 4 years old... not *that* ancient...

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.2
-  - 1.9.3
-  - 2.0.0
+  - 2.0
+  - 2.1
+  - 2.2
   - rbx-18mode
```
Comment (yob) [inline .travis.yml]: it's probably fine to remove support for rbx-18 and jruby-18 as well

**Comment by koriroys** [comment]
@yob I'm actually just trying to get it build on travis. Won't merge this. But it seems some dependencies require 1.9.3+

**Comment by koriroys** [comment]
I should say, some _development dependecies_ require 1.9.3+

**Comment by koriroys** [comment]
suspect the travis errors might be bundler related actually

**Comment by koriroys** [comment]
@yob Travis finally seems happy with this. What do you think?

**Comment by yob** [comment]
Does 2.2 pass if you add it as well?

**Comment by yob** [comment]
Does Travis have 2.3 preview1 available? If so, maybe it's worth adding it (with allow_failures) while you're changing the config

**Comment by koriroys** [comment]
@yob Yep, 2.2 and 2.3preview both fail. Seems to be some bundler/rubygems issue. I included 2.2 and 2.3preview with allow_failures, and will wait for travis to sort out the issue.

**Comment by yob** [comment]
Looks like the test suite requires test-unit or minitest, which were removed from the stdlib in 2.2.

I'll go ahead and merge this, and try to resolve that issue in a separate PR. We definitely want 2.2 to be a supported VM

**Pull request merged**